.github/workflows/cicd.yml

```diff
@@ -44,10 +44,6 @@ jobs:
       - name: Run tests
         run: |
           poetry run pytest --cov-report=xml
-      - name: Upload coverage to Codecov
-        uses: codecov/codecov-action@v1
-        with:
-          fail_ci_if_error: true
```

Comment:
This might mean that we won't be able to make a PR to the upstream 🤔 It is failing, because it expects some credentials in env vars probably, right?